include/conc/atomic.hpp

```diff
@@ -93,72 +94,121 @@ struct standard_policy {
 template <typename...> inline auto injected_policy = detail::standard_policy{};
 
 template <typename... DummyArgs, typename T>
+#if __cplusplus >= 202002L
     requires(sizeof...(DummyArgs) == 0)
+#endif
```

Comment:
suggestion: might be nice to wrap some of these things requiring C++20

Reply:
Good call, I made a macro.